src/domain/orders/details/index.tsx

```diff
@@ -0,0 +1,972 @@
+                    </div>
+                    <div className="inter-small-regular text-grey-50">
+                      {order?.currency_code.toUpperCase()}
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </BodyCard>
+            <BodyCard
+              className={"w-full mb-4 min-h-0 h-auto"}
+              title="Fulfillment"
+              status={<FulfillmentStatusComponent />}
+              customActionable={
+                <Button
+                  variant="secondary"
+                  size="small"
+                  onClick={() => console.log("Create")}
+                >
+                  Create Fulfillment
+                </Button>
+              }
+              actionables={[
+                {
+                  label: "Create fulfillment",
+                  onClick: () => console.log("Create"),
+                },
+              ]}
+            >
+              <div className="mt-6">
+                {order?.shipping_methods.map((method) => (
+                  <div className="flex flex-col">
+                    <span className="inter-small-regular text-grey-50">
+                      Shipping Method
+                    </span>
+                    <span className="inter-small-regular text-grey-90 mt-2">
+                      {method?.shipping_option.name || ""}
+                    </span>
+                    <div className="flex flex-col min-h-[100px] mt-8 bg-grey-5 px-3 py-2 h-full">
+                      <span className="inter-base-semibold">
+                        Data{" "}
+                        <span className="text-grey-50 inter-base-regular">
+                          (1 item)
+                        </span>
+                      </span>
+                      <div className="flex flex-grow items-center mt-4">
+                        <ReactJson
+                          name={false}
+                          collapsed={true}
+                          src={method?.data}
+                        />
+                      </div>
+                    </div>
+                  </div>
+                ))}
+                <div className="mt-6 inter-small-regular ">
+                  {allFulfillments.map((fulfillmentObj, i) => (
+                    <FulFillment key={i} fulfillmentObj={fulfillmentObj} />
+                  ))}
+                </div>
+              </div>
+            </BodyCard>
+            <BodyCard
+              className={"w-full mb-4 min-h-0 h-auto"}
+              title="Customer"
+              actionables={[
+                {
+                  label: "Edit Shipping Address",
+                  icon: <TruckIcon size={"20"} />,
+                  onClick: () =>
+                    setAddressModal({
+                      address: order?.shipping_address,
+                      type: "shipping",
+                    }),
+                },
+                {
+                  label: "Edit Billing Address",
```

Comment:
if there is no billing address this should probably be disabled, right now it shows as clickable

Reply:
or we should allow it to open with the shipping address options and if it's dirtied create a billing address for the order as well

Reply:
I like your idea here, but let's note it down for second round, if you are cool with that 😅

Reply:
Idea number two that is. Will add a fix to hide billing :D